cypress/support/utils/bpartner.js

```diff
@@ -220,3 +235,11 @@ function applyContact(bPartnerContact) {
   }
   cy.pressDoneButton();
 }
+
+// function applyBank(bPartner){
```

Comment:
hi @cristinastefan90 please remove this in one of your next commits. 
...not urgent, not need for a dedicated branch just so that it'S clean within the next days.
Or, if it should stay, pls write some comment about what to use it for (e.g. as a sortof template)

Reply:
ok, will do this in the next commit for the test I am working right now